Exercise-13/js/index.js

```diff
@@ -0,0 +1,39 @@
+import data from "../assets/friends.json" assert { type: "json" };
+
+const cardsContainer = document.getElementById("cardsContainer");
+
+function addFriendsData() {
+  const result = new DocumentFragment();
```

Comment:
could you use proper naming conventions instead of generic ones like result

Reply:
Ok, I will do that